### PR TITLE
Docsite: improve inventory-related pages

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Docsite: improve inventory-related pages

(Note:I know that two spaces between sentences is not considered as a mistake in US but imo would be good to use one approach, at least, throughout one file)

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
`docs/docsite/rst/plugins/inventory.rst`
`docs/docsite/rst/user_guide/intro_dynamic_inventory.rst`

### CODE REVIEW

**Comment by Andersson007** [comment]
ready_for_review

File: docs/docsite/rst/plugins/inventory.rst [hunk 1]
```diff
@@ -37,11 +37,11 @@ If the plugin is in a collection you need to use the fully qualified name:
 Using inventory plugins
 -----------------------
 
-To use an inventory plugin you need to provide an inventory source. Most of the time this is a file containing host information or a YAML configuration file with options for the plugin. You can use the ``-i`` flag to provide inventory sources or configure a default inventory path.
+To use an inventory plugin, you have to provide an inventory source. Most of the time this is a file containing host information or a YAML configuration file with options for the plugin. You can use the ``-i`` flag to provide inventory sources or configure a default inventory path.
```
Comment (acozine) [inline docs/docsite/rst/plugins/inventory.rst]: I agree with the spirit of this change - I think "must" is easier to translate. 
```suggestion
To use an inventory plugin, you must provide an inventory source. Most of the time this is a file containing host information or a YAML configuration file with options for the plugin. You can use the ``-i`` flag to provide inventory sources or configure a default inventory path.
```

**Review by acozine — COMMENTED** [review]
Thanks @Andersson007 for finding and fixing a bunch of errors in this PR! I found one nit to pick, otherwise it looks good.

File: docs/docsite/rst/plugins/inventory.rst (hunk 1 above)
Comment (Andersson007) [reply docs/docsite/rst/plugins/inventory.rst]: @acozine committed, thanks! will know

Comment (Andersson007) [reply docs/docsite/rst/plugins/inventory.rst]: ready_for_review

**Comment by acozine** [comment]
Thanks @Andersson007!

**Pull request merged**